package/npm/packages/elm-format-darwin-x64/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@avh4/elm-format-darwin-x64",
+  "version": "0.8.6-0",
+  "description": "The macOS 64-bit binary for elm-format.",
+  "repository": "https://github.com/avh4/elm-format",
+  "license": "BSD-3-Clause",
+  "os": [ "darwin" ],
+  "cpu": [ "x64", "arm64" ]
+}
```

Comment:
:thinking: does npm require this file to (and the README) to be in the repo somewhere?  I'm wondering if we could maybe just generate these folders when they're actually needed for publishing?

Reply:
We can generate them if you want. I just found it the easiest to have actual files so it’s possible to do manual fixups.

Not sure yet if _I_ will have time to do generation stuff though. A potential approach here is if you’d like to do that later in another PR.

Reply:
Sounds good!  I'm gonna take a look at generation on a new branch, then.